pwa-devdocs/src/technologies/basic-concepts/index.md

```diff
@@ -2,85 +2,85 @@
 
 ## Application shell
 
-An application shell provides the basic user interface structure for a progressive web application.    
+An application shell provides the basic user interface structure for a progressive web application.
 
 For more information, see [Application shell][].
 
 ## Service worker
 
 A service worker is a script that runs in the background.
-Progressive web applications use service workers for caching and resource retrieval.    
-<sub>[ *[Help write this topic][Service Worker]* ]</sub>
+Progressive web applications use service workers for caching and resource retrieval.  
+<sub>[ _[Help write this topic][service worker]_ ]</sub>
 
 ## Component data binding
 
 Component data binding refers to the way data flows between the source and a UI component.
-Progressive web applications use data binding patterns to connect dynamic data with the user interface.   
-<sub>[ *[Help write this topic][Component data binding]* ]</sub>
+Progressive web applications use data binding patterns to connect dynamic data with the user interface.  
+<sub>[ _[Help write this topic][component data binding]_ ]</sub>
 
 ## GraphQL
 
 GraphQL is a specification for a data query language client side and a service layer on the server side.
-It is used to request and push data in a progressive web application.   
+It is used to request and push data in a progressive web application.
 
 For more information, see [GraphQL][].
 
 ## CSS modules
 
 CSS modules are modular and reusable CSS styles.
-This allows you to develop components with styles that do not conflict with external style definitions.   
+This allows you to develop components with styles that do not conflict with external style definitions.
 
 For more information, see [CSS modules][].
 
 ## Client state, reducers, and actions
 
-Client state, reducers, and actions are [Redux] concepts used to manage and handle the state of a web application.  
-<sub>[ *[Help write this topic][Client state, reducers, and actions]* ]</sub>
+Client state, reducers, and actions are [Redux][] concepts used to manage and handle the state of a web application.  
+<sub>[ _[Help write this topic][client state, reducers, and actions]_ ]</sub>
 
 ## Loading and offline states
 
-Loading and offline are both states that must be handled by progressive web applications.   
-<sub>[ *[Help write this topic][Loading and offline states]* ]</sub>
+Loading and offline are both states that must be handled by progressive web applications.  
+<sub>[ _[Help write this topic][loading and offline states]_ ]</sub>
 
 ## Container extensibility
 
-Writing extensible containers allow others to re-use and alter your container without modifying the source. 
+Writing extensible containers allow others to re-use and alter your container without modifying the source.
 
-For more information, see [Container extensibility].
+For more information, see [Container extensibility][].
```

Comment:
This isn't rendering correctly in Markdown preview, and I'm having issues getting `npm run start` to run locally. Can you confirm this looks okay in rendered output (and maybe help me resolve my local error 😄 )